flow-typed/types/kaltura-player-options.js

```diff
@@ -7,7 +7,8 @@ declare type KPOptionsObject = {
   disableUserCache?: boolean,
   playback?: PKPlaybackConfigObject,
   sources?: PKSourcesConfigObject,
-  plugins?: PKPluginsConfigObject,
+  plugins: KPPluginsConfigObject,
```

Comment:
it is not optional anymore?

Reply:
it's {} by default

Reply:
We are initialise it internally, but from the app/user perspective this is optionally.